runtime/legion/legion_ops.cc

```diff
@@ -3158,6 +3213,7 @@ namespace Legion {
                                launcher.dst_requirements.size(), 
                              launcher.static_dependences,
                              launcher.predicate);
+      invoke_memoize_operation();
```

Comment:
Are we sure we want to do these calls in the initialization phase of the operations and not later once they are in the pipeline?